checks/normandy/uptake_error_rate.py

```diff
@@ -0,0 +1,132 @@
+"""
+The percentage of reported errors in Uptake Telemetry should be under the specified
+maximum.
+
+For each recipe whose error rate is above the maximum, the total number of events
+for each status is returned. The min/max timestamps give the datetime range of the
+dataset obtained from https://sql.telemetry.mozilla.org/queries/64921/
+"""
+from collections import defaultdict
+from typing import Dict, List
+
+import aiohttp
+
+from poucave.typings import CheckResult
+
+
+EXPOSED_PARAMETERS = ["max_error_percentage", "min_total_events"]
+
+REDASH_URI = (
+    f"https://sql.telemetry.mozilla.org/api/queries/64921/results.json?api_key="
+)
+
+# Normandy uses the Uptake telemetry statuses in a specific way.
+# See https://searchfox.org/mozilla-central/rev/4218cb868d8deed13e902718ba2595d85e12b86b/toolkit/components/normandy/lib/Uptake.jsm#23-43
+NORMANDY_STATUSES = {
+    "custom_1_error": "action_pre_execution_error",
+    "custom_2_error": "action_post_execution_error",
+    "server_error": "action_server_error",
```

Comment:
These are action statuses, but the rest of the check seems to focus on recipes. I'm not sure if you want to build another check for actions -- I think we'd see any action-specific failure across enough recipes to make it clear what was going on -- but if not, I think we should get rid of the action statuses. In particular, `custom_1_error` should be `RECIPE_ACTION_DISABLED`.